app-emulation/winetricks/winetricks-20170614.ebuild

```diff
@@ -21,7 +21,7 @@ SRC_URI="${SRC_URI}
 	kde? ( https://dev.gentoo.org/~tetromino/distfiles/wine/${wtg}.tar.bz2 )"
 
 DESCRIPTION="Easy way to install DLLs needed to work around problems in Wine"
-HOMEPAGE="http://winetricks.org http://wiki.winehq.org/winetricks"
```

Comment:
https://github.com/Winetricks/winetricks/blob/master/README.md states that https://github.com/Winetricks/winetricks is the home page, replacing the previous home page https://code.google.com/p/winetricks.

I can keep the Wiki page there too, but it doesn't look like it's official/considered a home page. I saw it as more of an extra documentation page.